<?xml version="1.0" encoding="UTF-8"?>

<ui version="4.0">
  <class>GUI_AuthClass</class>
  <widget class="QDialog" name="GUI_AuthClass">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>200</height>
      </rect>
    </property>
    <property name="maximumSize">
      <size>
        <width>400</width>
        <height>200</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Пожалуйста, авторизуйтесь</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
            <spacer name="horizontalSpacer_2">
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="loginAsGuestButton">
              <property name="text">
                <string>Войти как гость</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="horizontalSpacer">
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="OKButton">
              <property name="text">
                <string>Вход</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="CancelButton">
              <property name="text">
                <string>Отмена</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
            <widget class="QLabel" name="loginLabel">
              <property name="text">
                <string>Логин:</string>
              </property>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="QLineEdit" name="loginText" />
          </item>
          <item row="1" column="0">
            <widget class="QLabel" name="passwordLabel">
              <property name="text">
                <string>Пароль:</string>
              </property>
            </widget>
          </item>
          <item row="1" column="1">
            <widget class="QLineEdit" name="passwordText">
              <property name="echoMode">
                <enum>QLineEdit::Password</enum>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item row="0" column="0">
        <spacer name="verticalSpacer_2">
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>20</width>
              <height>40</height>
            </size>
          </property>
        </spacer>
      </item>
      <item row="3" column="0">
        <spacer name="verticalSpacer">
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>20</width>
              <height>40</height>
            </size>
          </property>
        </spacer>
      </item>
      <item row="2" column="0">
        <widget class="QLabel" name="inputErrorText">
          <property name="enabled">
            <bool>true</bool>
          </property>
          <property name="font">
            <font>
              <pointsize>10</pointsize>
            </font>
          </property>
          <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
          </property>
          <property name="autoFillBackground">
            <bool>false</bool>
          </property>
          <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);</string>
          </property>
          <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
            <string>Ошибка, вы ввели неверную комбинацию логина и пароля</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignCenter</set>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <tabstops>
    <tabstop>loginText</tabstop>
    <tabstop>passwordText</tabstop>
    <tabstop>OKButton</tabstop>
    <tabstop>CancelButton</tabstop>
    <tabstop>loginAsGuestButton</tabstop>
  </tabstops>
  <resources />
  <connections>
    <connection>
      <sender>CancelButton</sender>
      <signal>clicked()</signal>
      <receiver>GUI_AuthClass</receiver>
      <slot>reject()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>349</x>
          <y>175</y>
        </hint>
        <hint type="destinationlabel">
          <x>358</x>
          <y>359</y>
        </hint>
      </hints>
    </connection>
  </connections>
</ui>